examples/js/exporters/ColladaExporter.js

```diff
@@ -493,7 +499,15 @@ THREE.ColladaExporter.prototype = {
 
 					'</effect>';
 
-				libraryMaterials.push( `<material id="${ matid }" name="${ m.name }"><instance_effect url="#${ matid }-effect" /></material>` );
+				var materialNode = `<material id="${ matid }"` +
+					(
+						m.name ?
+							` name="${ m.name }` :
+							''
+					) +
+					`"><instance_effect url="#${ matid }-effect" /></material>`;
```

Comment:
I think there is a bug here. I the name is empty we'll end up with a double quote:

```xml
<material id="matid""><instance_effect url="#matid-effect" /></material>
```

Reply:
Oops I missed that -- thanks for double checking 😬

Reply:
I fixed and checked that in my project which uses reqiurejs so I have to wrap files with define, and is using standardjs which doesn't have semicolons, has spaces instead of tabs, so after I fixed it there I rewrote it in vim not to break project code style. And I introduced the bug there, oops. Sorry for not testing it afterwards, I would probably catch it if I did.